src/settings/PatronNotices/tokens.js

```diff
@@ -30,9 +30,12 @@ const formats = {
     'request.additionalInfo': 'Additional information regarding the request cancellation',
   },
   loan: {
-    'loan.dueDate': 'Dec 31, 2019 22:00',
-    'loan.initialBorrowDate': 'Jan 1, 2019 11:00',
-    'loan.checkedInDate': 'Dec 15, 2019 13:24',
+    'loan.dueDate': 'Dec 31, 2019',
+    'loan.dueDateTime': 'Dec 31, 2019 22:00',
+    'loan.initialBorrowDate': 'Jan 1, 2019',
+    'loan.initialBorrowDateTime': 'Jan 1, 2019 11:00',
+    'loan.checkedInDate': 'Dec 15, 2019',
+    'loan.checkedInDateTime': 'Dec 15, 2019 13:24',
```

Comment:
Would it make sense to provide these example date and time strings as `<FormattedDate>`, `<FormattedTime>`, etc? Likewise for other values (`fee.actionAdditionalInfo`) that contain translatable/localizable information?